tests/e2e/suite_test.go

```diff
@@ -33,24 +33,57 @@ import (
 )
 
 const (
-	samplesDir = "../../config/samples/"
+	samplesDir = "../../config/samples"
+	testConf   = "./conf"
 )
 
+const (
+	e2eNamespaceDrill    = "kubestone-e2e-drill"
+	e2eNamespaceFio      = "kubestone-e2e-fio"
+	e2eNamespaceIperf3   = "kubestone-e2e-iperf3"
+	e2eNamespacePgbench  = "kubestone-e2e-pgbench"
+	e2eNamespaceSysbench = "kubestone-e2e-sysbench"
+)
+
+var e2eNamespaces = []string{
```

Comment:
Would be nicer to create separate packages for each e2e test and let their init register
all the namespaces.

Reply:
created follow up ticket: https://github.com/xridge/kubestone/issues/134